sentry-uninstall.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd "$SCRIPT_DIR"
+
+NAMESPACE=common
+ENVIRONMENT="common/dev"  # Default environment
```

Comment:
Maybe better without default value, so the user of this script is more conscious of what environment he is uninstalling

Reply:
I will remove default one